ctypesgen/includes.py

```diff
@@ -0,0 +1,18 @@
+import re
+import mmap
+
+
+INCLUDE_RE = re.compile(rb'^\s*\#include\s+(["<][^">]+[">])', re.MULTILINE)
```

Comment:
See also https://stackoverflow.com/questions/1420017/regular-expression-to-match-c-include-file/